tests/training/metrics/test_fbeta_multi_label_measure.py

```diff
@@ -0,0 +1,291 @@
+from typing import List
+
+import pytest
+import torch
+from sklearn.metrics import precision_recall_fscore_support
+from torch.testing import assert_allclose
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.testing import AllenNlpTestCase, multi_device
+from allennlp_multi_label.training.metrics import FBetaMeasureMultiLabel
+
+
+class TestFBetaMeasureMultiLabel(AllenNlpTestCase):
+    def setup_method(self):
+        super().setup_method()
+        self.predictions = torch.tensor(
+            [
+                [0.55, 0.25, 0.10, 0.10, 0.20],
+                [0.10, 0.60, 0.10, 0.95, 0.00],
+                [0.90, 0.80, 0.75, 0.80, 0.00],
+                [0.49, 0.50, 0.95, 0.55, 0.00],
+                [0.60, 0.49, 0.60, 0.65, 0.85],
+                [0.85, 0.40, 0.10, 0.20, 0.00],
+            ]
+        )
+        self.targets = torch.tensor(
+            [
+                [1, 1, 0, 0, 0],
+                [0, 1, 0, 1, 0],
+                [1, 1, 0, 1, 0],
+                [1, 1, 1, 1, 0],
+                [1, 1, 1, 1, 0],
+                [0, 0, 0, 0, 0],
+            ]
+        )
+
+        # detailed target state
+        self.pred_sum = [4, 3, 3, 4, 1]
+        self.true_sum = [4, 5, 2, 4, 0]
+        self.true_positive_sum = [3, 3, 2, 4, 0]
+        # I don't think this is correct for multi-label
+        self.true_negative_sum = [10.0, 10.0, 12.0, 11.0, 14.0]
+        self.total_sum = [15, 15, 15, 15, 15]
+
+        # true_positive_sum / pred_sum
+        desired_precisions = [3 / 4, 3 / 3, 2 / 3, 4 / 4, 0 / 1]
+        # true_positive_sum / true_sum
+        desired_recalls = [3 / 4, 3 / 5, 2 / 2, 4 / 4, 0.00]
+        desired_fscores = [
```

Comment:
This is the assumption that beta = 1. We can rename the metric to be F1measuremultilabel

Reply:
Any reason to do that though? The code currently works with beta != 1. These unit tests assume beta is 1 yes, but so do Allen’s unit tests.

Reply:
yeah that's totally weird. I think this shouldn't be an issue since that's the case